server/routes/contacts/create/check-answers/createContactCheckAnswersController.test.ts

```diff
@@ -65,16 +66,12 @@ describe('GET /prisoner/:prisonerNumber/contacts/create/check-answers/:journeyId
 
     // Then
     expect(response.status).toEqual(200)
-    expect(response.text).toContain('Contacts')
-    expect(response.text).toContain('Hmpps Contacts Ui')
-    expect(response.text).toContain('Check your answers')
-    expect(auditService.logPageView).toHaveBeenCalledWith(Page.CREATE_CONTACT_CHECK_ANSWERS_PAGE, {
-      who: user.username,
-      correlationId: expect.any(String),
-    })
```

Comment:
I refactored these tests to separate the checking of page content and checking of the audit message